src/test/resources/alma/HT016709661.json

```diff
@@ -12,6 +12,16 @@
       "label" : "lobid Organisation"
     },
     "id" : "https://lobid.org/item/990184127410206441"
+  }, {
+    "id" : "https://lobid.org/item/53482457280006441",
+    "type" : [ "Item" ],
+    "label" : "Portfolio",
+    "heldBy" : {
+      "id" : "http://lobid.org/organisations/DE-605#!",
+      "label" : "lobid Organisation"
```

Comment:
Here it's always DE-605, while at least one entry in the XML indicates "Düsseldorf" and "Dortmund(?)" while having an ID for DE-605.

Reply:
(mention this in https://github.com/hbz/lobid-resources/issues/1135)